coniferest/session/callback.py

```diff
@@ -50,3 +54,72 @@ def __call__(self, metadata, data, session) -> None:
         self.iteration += 1
         if self.iteration >= self.budget:
             session.terminate()
+
+
+def autonomous_decider_callback(metadata: Label, data: np.ndarray, session: 'Session') -> Label:
+    "Use metadata as already defined labels for making decision."
+    return metadata
+
+
+class AutonomousExperiment:
```

Comment:
The purpose of this class is not clear from the name and the docstring. I don't understand the name, but I cannot think about anything better. @matwey, do you have better ideas?
More documentation will be useful, but could go to another PR.